File: src/server/BuildingBlock/MovieTicket.Domain/Entities/ServiceReservation.cs
using MovieTicket.Core.Domain;

namespace MovieTicket.Domain.Entities;

public class ServiceReservation :  EntityBase
{
    public Guid ReservationId { get; set; }
    public virtual Reservation Reservation { get; set; }
    
    public Guid ServiceId { get; set; }
    public virtual Service Service { get; set; }
    
    public int Quantity { get; set; }
    public Double Price { get; set; }
}

File: src/client/mobile/app/src/main/java/com/superman/movieticket/domain/entities/ServiceReservation.kt
package com.superman.movieticket.domain.entities

import com.superman.movieticket.core.entity.BaseEntity
import java.io.Serializable

data class ServiceReservation (
    var reservationId: String,
    var reservation: String,
    var serviceId: String,
    var service: Service,
    var quantity: Int,
    var price: Long
) : BaseEntity(), Serializable